eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml

```diff
@@ -18,9 +18,6 @@ parameters:
   - name: DotNetCoreVersion
     type: string
     default: ''
-  - name: SkipDotNetInstall
```

Comment:
Where there any usages of this?

Reply:
@weshaggard yes, but they were unnecessary, and so I removed them in this PR:
- #4937